misk/src/main/kotlin/misk/web/jetty/WebActionsServlet.kt

```diff
@@ -75,6 +80,9 @@ internal class WebActionsServlet @Inject constructor(
       }
     } catch (_: ProtocolException) {
       // Probably an unexpected HTTP method. Send a 404 below.
+    } catch (e: Throwable) {
+      log.error { "Uncaught exception on ${request.dispatchMechanism()} ${request.httpUrl()}" }
```

Comment:
Should this also log the exception itself?

Reply:
Yeah good catch